code/datums/jobs/job/job.dm

```diff
@@ -10,7 +10,6 @@ GLOBAL_LIST_INIT(exp_jobsmap, list(
 
 GLOBAL_LIST_INIT(exp_specialmap, list(
 	EXP_TYPE_LIVING = list(),
```

Comment:
Do we store marine vs xeno exp time?

Reply:
Honestly I haven't messed with the XP system much, it was LaK's interest.
Do you want something changed?

Reply:
No just looked underrepresented.